src/components/Nav.tsx

```diff
@@ -149,14 +153,22 @@ export const NavHeader = () => {
       boxShadow="0 6px 0px 0px #005493" //blue
     >
       <Flex alignItems="center">
-        <Box mr="15px" bg="gray.100" h="40px" w="40px" borderRadius="50%">
-          <Image
-            src={`${process.env.PUBLIC_URL}/logo.png`}
-            alt="schedulator logo"
-            minWidth="40px"
-            padding="5px"
-          ></Image>
-        </Box>
+        <Link2
+          to={
+            user?.roles.includes("admin") || user?.roles.includes("user")
+              ? "/login"
+              : "/"
```

Comment:
Is it just me or is this condition backwards? We want it to go to `/login` when there is no user, not when there is a user.

Reply:
when logged in, navigation to /login will redirect you to the dashboard if you're an admin and preference survey if you're a regular user. If you're not logged in clicking the logo brings you to the home page not the login page

Reply:
Did we want to just route admins directly to /dashboard, users to /survey, and not logged in to / ? its kinda odd seeing the login page show up for half a second before it redirects

Reply:
Yep, sounds good!